Documentation/tuning.md

```diff
@@ -18,18 +17,22 @@ This timeout is how long a follower node will go without hearing a heartbeat bef
 By default, etcd uses a `1000ms` election timeout.
 
 Adjusting these values is a trade off.
-Lowering the heartbeat interval will cause individual commands to be committed faster but it will lower the overall throughput of etcd.
-If your etcd instances have low utilization then lowering the heartbeat interval can improve your command response time.
+A good option for heartbeat interval is in the range [0.5*RTT, 1.5*RTT], in which RTT represents average round-trip time between nodes.
+If heartbeat interval is low, etcd may use amounts of CPU and network resources to maintain the heartbeat.
```

Comment:
If heartbeat interval is too low, etcd might use unnecessary CPU and network resources.